test/core/surface/channel_init_test.cc

```diff
@@ -0,0 +1,172 @@
+// Copyright 2023 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "src/core/lib/surface/channel_init.h"
+
+#include <algorithm>
+#include <map>
+#include <string>
+
+#include "absl/strings/string_view.h"
+#include "gtest/gtest.h"
+
+#include "src/core/lib/channel/channel_stack.h"
+#include "src/core/lib/channel/channel_stack_builder_impl.h"
+#include "test/core/util/test_config.h"
+
+namespace grpc_core {
+namespace {
+
+const grpc_channel_filter* FilterNamed(const char* name) {
```

Comment:
Suggest calling this something like `CreateFilter()`.

Reply:
like the other test... i think this form ends up making the actual tests more readable by focusing the reader on the pertinent thing